common/app/layout/Sublink.scala

```diff
@@ -274,8 +276,28 @@ case class ContentCard(
   timeStampDisplay: Option[FaciaCardTimestamp],
   shortUrl: Option[String],
   useShortByline: Boolean,
-  group: String
+  group: String,
+  branding: Option[Branding] = None,
```

Comment:
I wouldn't think this constructor is called in many places. Could we remove the defaults and replace it with explicit `None`s where it gets called?